config/samples/httpbin_blacklist.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: httpbin
```

Comment:
It might be better to rename it something like `httpbin-blacklist` along with `spec.name` field.